tools/eval.py

```diff
@@ -29,8 +29,18 @@
 from torchreid.utils import (Logger, set_random_seed,
                              load_pretrained_weights, get_model_attr)
 from torchreid.engine import build_engine
+from torchreid.integration.nncf.compression import wrap_nncf_model, is_checkpoint_nncf
 
 
+def make_nncf_changes_in_eval(model, cfg):
```

Comment:
Kirill, are you sure? If I move this function, the part of logic will be hidden here.
IMHO it is better to keep the functions `make_nncf_changes_in_training` in `tools/main.py` and the function `make_nncf_changes_in_eval` in `tools/eval.py`

I think I will move the function `calculate_lr_for_nncf_training`, since it may be isolated from other code, but I would prefer to keep `make_nncf_changes_in_training`.

Reply:
After comments @sovrasov moved it to a new file.